src/NumericalAlgorithms/DiscontinuousGalerkin/MetricIdentityJacobian.hpp

```diff
@@ -183,4 +183,46 @@ void metric_identity_jacobian(
     const tnsr::I<DataVector, Dim, Frame::Inertial>& inertial_coords,
     const Jacobian<DataVector, Dim, Frame::Logical, Frame::Inertial>&
         jacobian) noexcept;
+
+/*!
+ * \ingroup DiscontinuousGalerkinGroup
+ * \brief Compute the Jacobian. inverse Jacobian, and determinant of the
+ * Jacobian so that they satisfy the metric identities.
+ *
+ * Uses `metric_identity_jacobian` to compute the determinant of the Jacobian
+ * times the inverse Jacobian. We can then compute the determinant of the
+ * Jacobian in \f$d\f$ spatial dimensions as
+ *
+ * \f{align}{
+ *
+ *  &J =
+ *   \left\{
+ *   \begin{array}{ll}
+ *     \sqrt[(d-1)]{J^{d-1}} & \mathrm{if} \; d \; \mathrm{even} \\
+ *     \mathrm{sgn}\left(\frac{J^{-1}\partial_\xi x}{\partial_\xi
+ *     x}\right)\sqrt[(d-1)]{J^{d-1}} & \mathrm{if} \; d \; \mathrm{odd}
+ *   \end{array}\right.
+ *
+ * \f}
+ *
+ * From this we can compute the inverse Jacobian and then invert it to obtain
+ * the Jacobian that is exactly the inverse of the inverse Jacobian.
+ *
+ * \warning on entry `Jacobian` must be the Jacobian to use for compute the
+ * determinant of the Jacobian times the inverse Jacobian so that it satisfies
+ * the metric identities. On output the Jacobian is the inverse of the inverse
+ * Jacobian that satisfies the metric identities.
+ */
+template <size_t Dim>
+void metric_identity_jacobian_and_inv_jacobian(
```

Comment:
1. I don't see why the order you suggest is the preferable one. I would argue that `det_jac_times_inverse_jacobian`, `det_jacobian`, `inverse_jacobian`, `jacobian` is the preferable order because this is the order things are computed in (up to possibly Jacobian & inverse Jacobian). Additionally, having `jacobian` at the end of the list puts it closest to the input arguments which I think would be reasonable given it must be specified on input.
2. sure!

Reply:
Oh, option 1. was aiming to match the args to the name of the function, but otherwise I agree with you. Plus I see you've done option 2., so I'm satisfied 👍